ccmlib/scylla_node.py

```diff
@@ -505,12 +506,19 @@ def _update_jmx_pid(self, wait=True):
         start = time.time()
         while not (os.path.isfile(pidfile) and os.stat(pidfile).st_size > 0):
             if time.time() - start > 30.0 or not wait:
-                print_("Timed out waiting for pidfile to be filled "
-                       "(current time is %s)" % (datetime.datetime.now()))
-                return
+                print_("Timed out waiting for pidfile {} to be filled (after {} seconds): File {} size={}".format(
+                        pidfile,
+                        0 if not wait else time.time() - start,
+                        'exists' if os.path.isfile(pidfile) else 'does not exist' if not os.path.exists(pidfile) else 'is not a file',
+                        os.stat(pidfile).st_size if os.path.exists(pidfile) else -1))
+                break
```

Comment:
the switch from `return` to `break`, is that on purpose here ?

Reply:
Yes, since the `wait` case is handled outside the loop now, and the timeout case is handled consistently like the other similar loops.